tests/testthat/test-ReportCard.R

```diff
@@ -39,6 +39,13 @@ testthat::test_that("append_plot accepts a ggplot", {
   )
 })
 
+testthat::test_that("append_plot accepts a ggplot with a dim", {
+  testthat::expect_error(
+    ReportCard$new()$append_plot(ggplot2::ggplot(iris), c(1000L, 100L)),
+    regexp = NA
+  )
+})
+
```

Comment:
please add a test case with no dim added (what is the expected output) and with wrong dim input.